governance/xc_admin/packages/xc_admin_frontend/components/tabs/Proposals.tsx

```diff
@@ -0,0 +1,411 @@
+import { BN, Wallet } from '@coral-xyz/anchor'
+import { PythCluster } from '@pythnetwork/client/lib/cluster'
+import { useAnchorWallet } from '@solana/wallet-adapter-react'
+import { AccountMeta } from '@solana/web3.js'
+import { getIxPDA } from '@sqds/mesh'
+import { MultisigAccount, TransactionAccount } from '@sqds/mesh/lib/types'
+import copy from 'copy-to-clipboard'
+import { useRouter } from 'next/router'
+import {
+  Dispatch,
+  SetStateAction,
+  useCallback,
+  useContext,
+  useEffect,
+  useState,
+} from 'react'
+import {
+  getProposals,
+  MultisigParser,
+  PythMultisigInstruction,
+  WormholeMultisigInstruction,
+} from 'xc_admin_common'
+import { ClusterContext } from '../../contexts/ClusterContext'
+import { SECURITY_MULTISIG, useMultisig } from '../../hooks/useMultisig'
```

Comment:
Maybe allow people that don't connect their wallet to see the proposals. 
You can do that by changing `useMultisig` for people that haven't connect their wallet.